kimchi/src/circuits/lookup/lookups.rs

```diff
@@ -37,11 +37,13 @@ pub struct LookupInfo {
     pub max_per_row: usize,
     /// The maximum joint size of any joint lookup in a constraint in `kinds`. This can be computed from `kinds`.
     pub max_joint_size: u32,
+    /// True if runtime lookup tables are used.
+    pub uses_runtime_tables: bool,
```

Comment:
this looks unnecessary, but I like it. But am I wrong in saying that this is unnecessary?

Reply:
We need this on the proof-systems side, to be able to turn on `uses_runtime_tables` in the generated plonk scalar calculations.

Reply:
I meant that this information is also present in the lookup index (if the Option is set for runtime table)